pages/_error.tsx

```diff
@@ -0,0 +1,34 @@
+import { NextPageContext } from 'next'
```

Comment:
So unlike the 404 page Next offers both [a static 500](https://nextjs.org/docs/advanced-features/custom-error-page#customizing-the-500-page) option and [a more flexible one](https://nextjs.org/docs/advanced-features/custom-error-page#more-advanced-error-page-customizing). We're going with the second one and clobbering it a little bit, returning the same component regardless of what the status code is.